products/opensuse/main.pm

```diff
@@ -314,6 +314,21 @@ elsif (get_var('SECURITY_TEST')) {
     prepare_target();
     load_security_tests;
 }
+elsif (get_var('VIRT_AUTOTEST')) {
+    prepare_target() unless get_var('IPMI_DO_NOT_RESTART_HOST');
+    loadtest "virt_autotest/switch_to_ssh_and_install_hypervisor";
+    loadtest "virt_autotest/reboot_and_wait_up_normal" unless get_var('IPMI_DO_NOT_RESTART_HOST');
+    loadtest "virt_autotest/uefi_guest_installation"   unless get_var("SKIP_GUEST_INSTALL");
```

Comment:
please use consistent single-ticks unless you want to evaluate variables. In the line above you use single ticks here again double ticks. This comment applies to *all* changes, not just this line

Reply:
Thanks. Done